index.html

```diff
@@ -232,5 +232,49 @@ <h1 class="sectionheading">About Myself</h1>
         </ul>
       </div>
     </section>
+    <section class="contactform-container">
+      <form
+        action="https://formspree.io/f/mwkdkrga"
+        method="post"
+        class="contactform"
+      >
+        <div class="contactheading">
+          <h1 class="c-h">Contact me</h1>
+          <p class="contactpara">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it!
+          </p>
+        </div>
+
+        <input
+          class="nameinput"
+          type="text"
+          name="name"
+          id="username"
+          placeholder="Lorem Ipsum"
+          required
+        />
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that **Name field accepts up to 30 chars**, Kindly Fix it.